src/symbolize.h

```diff
@@ -80,7 +80,7 @@
 #ifndef GLOG_NO_SYMBOLIZE_DETECTION
 #  ifndef HAVE_SYMBOLIZE
 // defined by gcc
-#    if defined(HAVE_ELF_H) || defined(HAVE_SYS_EXEC_ELF_H)
+#    if defined(HAVE_ELF_H) || defined(HAVE_SYS_EXEC_ELF_H) || defined(HAVE_LINK_H)
 #      define HAVE_SYMBOLIZE
```

Comment:
thanks for letting me know this. i agree this would probably introduce undesirable side effects. 

In the recent version of glog, HAVE_SYMBOLIZE is not "derived" anymore on linux platform with bazel, unlike previous versions. (i am referring to the one we were using from jan-02-2024)

what's your suggestion getting around with that? just hardcode `-DHAVE_SYMBOLIZE` in `linux_only_copts` inside glog.bzl? If so, do you think if anybody else needs this (Im fine (not) sending the patch)

thnx